src/schema/v2/partner.ts

```diff
@@ -200,7 +204,11 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
             has_published_artworks: args.hasPublishedArtworks,
           }
 
-          return allViaLoader(partnerArtistsForPartnerLoader, {
+          // If we are authenticated / loader exists use the all loader
```

Comment:
Is that correct? The 'all' loader will exist for all requests that come from a logged-in user (since that will include the `X-Access-Token` header, and the presence of the header is what Metaphysics considers as 'authenticated' for loader purposes in https://github.com/artsy/metaphysics/blob/594022480e18cf881ad215c404855c3f0d38acda/src/lib/loaders/index.ts#L82-L84 - there's no verification happening).

So that means if this field is being used in Force/Eigen somewhere, with this change the authenticated loader will be used for all logged-in users, which might then fail as gravity enforces verification on that endpoint: https://github.com/artsy/gravity/blob/db8d902085f39a13d2a888611aeddbe35d16bffa/app/api/v1/partners_artists_endpoint.rb#L99

Reply:
thanks @mzikherman! that makes sense and this felt sketchy while I was writing it, thank you for clarifying how this works 🙌 
I think instead then I will need a separate endpoint on partner for the all loader? is that what you would suggest?

Reply:
I think I will take a page from the artwork endpoint and add an explicit param to use this loader 🤔 
`artwork(id: "test-7", includeUnlisted:  true)`
maybe `includePartnerFields` or something, let me know if it doesn't make sense